### ISSUE TITLE
Does this still work?

### ISSUE THREAD

**Issue by dmnchild** [APushingBoy/qsv-to-mp4#1]
Greetings, 
Been trying to find a solution to convert iq/iqiyi.com files from qsv format to mp4. Seems after loading the file, when I hit convert it doesnt do anything. No error, no convert.

**Comment by APushingBoy** [comment]
Yes, this still works. I used this tool several days ago. Can you specify the problem you encountered?

**Comment by dmnchild** [comment]
I used phone to get the qsv file for a show, moved to computer, imported via app and it shows up in there, hit the concert button, and nothing at all happened. Since no real instructions and me not reading/speaking Chinese I may have overlooked something. But it seems pretty self explanatory: too button import/load file, second clears, third starts, fourth stops (if it starts I assume), bottom info about you. Then there’s the drop down which I’m guessing default is output to same directory, and if you choose bottom you can specify. Left bottom obviously output preference.

**Comment by APushingBoy** [comment]
What you say about the buttons and the default output directory are all right, so it should work properly. Can you send your qsv file to me so I can try to convert it? You can share it through google drive.

**Comment by dmnchild** [comment]
https://drive.google.com/file/d/1NxIYsnc9HvnI-FX9iQX_puHqKMpMo2yG/view?usp=sharing

**Comment by dmnchild** [comment]
If it does work for you, possible OS difference? I am on windows 11 unfortunately. (also delete that comment or let me know to delete when finished;p)


edit: yesterday it just didnt do anything when i clicked the convert, today its giving me app error when trying to load the file. (even after a re download)

**Comment by APushingBoy** [comment]
> If it does work for you, possible OS difference? I am on windows 11 unfortunately. (also delete that comment or let me know to delete when finished;p)
> 
> edit: yesterday it just didnt do anything when i clicked the convert, today its giving me app error when trying to load the file. (even after a re download)

I'll try to convert your qsv file and then give you a feedback. I'm using windows 11 too, and I don't think OS is the key to the problem cause as I know windows 11 is just win10 with some new fashioned UI lol(Microsoft, you lazy guys)

**Comment by APushingBoy** [comment]
I tried your qsv file and I failed too. I think the reason is that you download the video to your phone and then move the qsv file to computer. As I know, the downloaded file(e.g. when you download video) of iqiyi on Android phone consists of not only the qsv file, but also some other files about the infomation of the video, and the qvs file's name is just in the form of '253590001_17935739000.qsv'. In this case, the infomation file is necessary, although iqiyi can play the single qsv file. 
While my case is that I'm using iqiyi uwp(China version)downloaded from microsoft store, whose downloaded file is just a qsv file named the video's name(such as 'Frozen II [1080p]'. And these qsv files can be converted.
So I suggest that you download the video using a windows client rather than using your phone.

**Comment by dmnchild** [comment]
Ah ok. and you are correct. I was using android app. Doesnt seem to be an english version of the app for me to try (Trying to fumble around the chinese version to see if i can login and test!) thanks for the help, well see if i can figure something out!

**Comment by big-Totoro6** [comment]
我试试
